scripts/itkdev-docker-compose

```diff
@@ -104,6 +105,26 @@ EOF
   fi
 }
 
+function enabledNFS {
+  echo "Setting up nfs..."
+  U=`id -u`
+  G=`id -g`
+
+  LINE="/Users -alldirs -mapall=$U:$G localhost"
```

Comment:
Using `$HOME` rather that `/Users` will probably be more correct here.

Reply:
Nope, it should be `/System/Volumes/Data/Users/` for performance and macs fake /Users